.github/workflows/gradle_build.yml

```diff
@@ -0,0 +1,41 @@
+# This workflow ensures the building step works
+#
+# @author Armin Schnabel
+# @version 1.0.0
+# @since 5.0.0-beta4
+name: Gradle Build
+
+on:
+  push:
+    branches:
+      - master
+      - dev
+  pull_request:
+    branches:
+      - master
+      - dev
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v1
+      - name: Set up JDK 1.8
+        uses: actions/setup-java@v1
+        with:
+          java-version: 1.8
+
+      - name: Create empty truststore file
+        run: |
+          # When no truststore is required by the server we create an empty file or else the build fails
+          mkdir -p synchronization/src/main/res/raw
+          touch synchronization/src/main/res/raw/truststore.jks
+
+      - name: Create local.properties with a read token
+        run: |
+          # Use a personal read token to install the Cyface Utils package
+          echo "github.user=${{ secrets.GITHUB_READ_ACCOUNT }}" > local.properties
+          echo "github.token=${{ secrets.GITHUB_READ_TOKEN }}" >> local.properties
+      - name: Build with Gradle
+        run: ./gradlew build
```

Comment:
Is it possible to run the tests here as well?

Reply:
The UnitTests are already executed by the "Build with Gradle" step (https://github.com/cyface-de/android-backend/commit/1fea91627bfcf1c9814bd454332d0dac05de302f/checks?check_suite_id=389946445).

The Integration test require an emulator on Github Actions.
I created a Ticket for checking out if this is (easily) possible: 
https://cyface.atlassian.net/browse/CY-5074